dice-game-ui-angular/src/app/saved-games/saved-games.component.html

```diff
@@ -0,0 +1,24 @@
+<h1 align="center" class="text-muted">Saved games</h1>
+
+<div *ngIf="games.length>0; else noSavedGames">
+    <mat-card class="m-1" *ngFor="let ge of games">
+        <mat-card-header>
+            <mat-card-title>
+                <button mat-flat-button [routerLink]="['/game', ge.guid]">
+                    Single game {{ge.gameEngine.fieldSize.width}} x {{ge.gameEngine.fieldSize.height}} from {{ge.creationDate|date:"dd.MM.yyyy"}}
```

Comment:
And it is opposite here - single quotes, because it is TS code, not html.

Reply:
Got it. Fixed.